packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -150,6 +150,26 @@ class FileCollection extends DocumentCollection {
     return this.stackClient.fetchJSON('POST', uri`/files/trash/${id}`)
   }
 
+  /**
+   * async definitelyDeleteFile - Definitely delete a file
```

Comment:
deleteFilePermanently is better IMHO. :)

Reply:
it would have been an awful last PR so I fixed it ;-)

Reply:
Thanks !